question_answering/item.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
```

Comment:
?
You should have changed the `version` not the `apiVersion`. also `mlrunVersion` is old too, needs to be updated as well.